week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -21,7 +21,15 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 */
 //1)Create the function inside, Think about nested loops
 function sortAges(arr) {
-  //create function here
+  let newArr = [];
+  for ( i in arr ) {
+    if ( typeof arr[i] == "number" ){
+      newArr.push(arr[i]);
+    }
+  }
+  newArr = newArr.sort(function(a, b){return a-b});
+  return newArr

```

Comment:
It's right! Just an extra tip:
You can sort the array in-line with the return statement, don't need to reassign it to a variable if you don't use it anymore.
As a result, you can simply comprise into one line:
`return newArr.sort(function(a, b){return a-b});`